pkg/api/v1/types.go

```diff
@@ -710,6 +710,8 @@ type ISCSIVolumeSource struct {
 	IQN string `json:"iqn"`
 	// iSCSI target lun number.
 	Lun int `json:"lun"`
+	// Optional: Defaults to 'default' (tcp). iSCSI interface name that uses an iSCSI transport.
```

Comment:
You can refer to the the [open-iscsi documentation Section 5.1.1](http://www.open-iscsi.org/docs/README), the [openstack documentation](http://docs.openstack.org/kilo/config-reference/content/iscsi-iface-config.html). An open-iscsi interface is at minimum a 3-tuple of transport name, iface name and hardware address (MAC) that will direct iscsi traffic over a particular HBA/network interface/offload driver instead of going over the default network interface.